examples/ownable-two-step/tests/ownable_two_step.rs

```diff
@@ -151,6 +153,113 @@ async fn accept_ownership(alice: Account, bob: Account) -> Result<()> {
     Ok(())
 }
 
+#[e2e::test]
+async fn cancel_transfer_ownership(alice: Account, bob: Account) -> Result<()> {
```

Comment:
I think this name is better. I don't think we have a written rule for how to name tests yet, but personally I prefer to prefix them with the name of the function. And this is the convention I followed on this file
```suggestion
async fn transfer_ownership_cancel_transfer(alice: Account, bob: Account) -> Result<()> {
```